pkg/spdk/server.go

```diff
@@ -46,6 +48,11 @@ func NewServer(ctx context.Context, portStart, portEnd int32) (*Server, error) {
 		return nil, err
 	}
 
+	if _, err = cli.BdevNvmeSetOptions(helpertypes.DefaultCtrlrLossTimeoutSec, helpertypes.DefaultReconnectDelaySec,
```

Comment:
nit: For better formatting, you can write this way instead.

```go
err = cli.BdevNvmeSetOptions(
    helpertypes.DefaultCtrlrLossTimeoutSec, 
    helpertypes.DefaultReconnectDelaySec, 
    helpertypes.DefaultFastIOFailTimeoutSec, 
    helpertypes.DefaultTransportAckTimeout,
)
if err != nil {
    return nil, errors.Wrap(err, "failed to set nvme options")
}
```

Reply:
Updated, thank you.